spec/app_spec.rb

```diff
@@ -62,5 +62,14 @@
 
       expect(last_response.body).to include(email)
     end
+
+    it 'has a logout link back to root' do
+      get '/auth/result', code: code
+
+      doc = Nokogiri::HTML(last_response.body)
+
+      logout_link = doc.at('a[href="/"]')
+      expect(logout_link.text.strip).to eq('Log out')
```

Comment:
So this is just checking the rendered page, not following it and visiting the page...I guess I could do a `get` to that location and then make sure the email isn't there anymore....will update momentarily